frontend/src/API.js

```diff
@@ -116,6 +117,30 @@ export const scoreResult = async ({
     }
 };
 
+export const scoreIntermediateResult = async ({
+    session,
+    participant,
+    result,
+}) => {
+    try {
+        const vars = {
+            session_id: session.id,
+            json_data: JSON.stringify(result),
+            csrfmiddlewaretoken: participant.csrf_token
+        };
+
+        const response = await axios.post(
+            API_BASE_URL + URLS.result.intermediateScore,
+            qs.stringify(vars)
+        );
+        return response.data;
+    } catch (err) {
+        console.error(err);
+        return null;
+    }
```

Comment:
Wouldn't it be better to leave the error handling to the component in question? Perhaps `MatchingPairs` wants to handle an error differently than another component.

Reply:
I followed the pattern of most API functions right now: return null and log the error to the console. It's the responsibility of components to display an error to the user. Throwing the error here gives less useful output to a participant. We should display a message that something went wrong, and developers can see the full error message via Sentry.